### PR TITLE
Create bug-reporting-template.md

### PR DESCRIPTION
**Pull request by calina-c**

### CODE REVIEW

File: bug-reporting-template.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#### Environment / database:
+#### Logged in user and password (if applicable):
+#### Starting URL / command executed:
+#### Steps to reproduce:
+
+* step 1:
+  * Form data (if applicable):
+* step 2
+	* Form data (if applicable):
```
Comment (stocarul) [inline bug-reporting-template.md]: Could you please check the indentation here?

**Comment by calina-c** [comment]
@stocarul I've fixed the indentation. It had no effect on the view, but indeed it is more coherent now.

**Comment by stocarul** [comment]
Thank you @calina-c. One quick questions: shouldn't we have a link to it? (from `bug-reporting.md`). And maybe from `README.md` to `bug-reporting.md`?

**Comment by calina-c** [comment]
@stocarul updated

**Review by stocarul — APPROVED** [review]
Awesome! 💯

**Pull request merged**